mapslice_test.go

```diff
@@ -50,3 +50,33 @@ func TestUnmarshal(t *testing.T) {
 		t.Errorf("expected: %s\ngot: %s", e, r)
 	}
 }
+
+func ExampleMapSlice_MarshalJSON() {
```

Comment:
I would put these in a separate test file with `package mapslice_test` that way you would have to import `mapslice`. This would serve better as an example.

Reply:
For Go, I think this is fairly idiomatic, and the example that will show in the doc.  https://go.dev/blog/examples

Reply:
It's a nitpick but consider a beginner that want to copy/paste it won't work out of the box.

Reply:
I'm fine with not doing it just giving you the reason I mentioned it.

Reply:
I'll include a link in the doc to the Go playground with a copy/pasteable.  Best of both worlds.

Reply:
@ake-persson, I came across something from Joe Tsai that is what I think you wanted.  I removed the `_example` folder, and created file `example_test.go`, with `package mapslice_test`.  The tests will appear in the docs and they import package `mapslice-json` as requested.  

I think that covers everything.  👍